demos/01 Unsupervised Classification/01c Interactive Deployment with Dynamic Compute.ipynb

```diff
@@ -172,7 +172,13 @@
     "    bands,\n",
     "    start_datetime=\"2023-06-01\",\n",
     "    end_datetime=\"2023-09-01\",\n",
-    ").filter(lambda x: x.cloud_fraction < 0.1)"
+    ")\n",
+    "try:\n",
+    "    # this is the <1.3.0 dynamic-compute version\n",
+    "    s2_stack = s2_stack.filter(lambda x: x.cloud_fraction < 0.1)\n",
+    "except Exception:\n",
+    "    # this is the >=1.3.0 dynamic-compute version\n",
+    "    s2_stack = s2_stack.filter(p.cloud_fraction < 0.1)"
```

Comment:
You may not have seen this, but @stephencpope suggested in [slack](https://descartes.slack.com/archives/C01AUJWSG4A/p1728605510515239) that we reverse this try/except such that the latest version goes in the try, and we catch for the older dc version

```suggestion
    ")\n",
    "try:\n",
    "    # this is the >=1.3.0 dynamic-compute version\n",
    "    s2_stack = s2_stack.filter(p.cloud_fraction < 0.1)",
    "except Exception:\n",
    "    # this is the <1.3.0 dynamic-compute version\n",
    "    s2_stack = s2_stack.filter(lambda x: x.cloud_fraction < 0.1)"
```

Reply:
give me a second to test this -- I'm not actually sure the old method throws an exception